botocore/httpchecksum.py

```diff
@@ -432,6 +435,14 @@ def _apply_request_trailer_checksum(request):
     )
 
 
+def _register_checksum_algorithm(algorithm):
+    checksum_algorithm_name = algorithm["algorithm"]
+    checksum_algorithm_name_feature_id = (
+        "FLEXIBLE_CHECKSUMS_REQ_" + checksum_algorithm_name.upper()
```

Comment:
We'll need to handle the `CRC64NVME` case since the feature id associated with it used `CRC64`.